pai-management/doc/how-to-write-pai-configuration.md

```diff
@@ -211,6 +210,9 @@ hadoop:
   custom-hadoop-binary-path: None
   hadoop-version: 2.7.2
   virtualClusters:
+    default:
```

Comment:
It is fine; but actually, the ‘default’ virtual cluster does not to be explicitly specified here. :)

Reply:
I would like to include a "typical" case where default is present. As our VC access policy grants default queue to everyone (and our e2e test deployment will use default queue). While if other people don't set default in the document, they will running into deployment issues .

---
In reply to: [195369291](https://github.com/Microsoft/pai/pull/727#discussion_r195369291) [](ancestors = 195369291)